scripts/js/commands/checkInternalLinks.ts

```diff
@@ -106,6 +106,7 @@ const RUNTIME_GLOBS_TO_LOAD = [
   "docs/guides/*.{mdx,ipynb}",
```

Comment:
This file should be closely reviewed. It's necessary because this is our first time having historical versions for qiskit-transpiler-service